lib/stat_tracker.rb

```diff
@@ -0,0 +1,34 @@
+require 'CSV'
+
+class StatTracker
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(games, teams, game_teams)
+    @games = games
+    @teams = teams
+    @game_teams = game_teams
+  end
+
+  def self.from_csv(locations_params)
+    games_file = locations_params[:games]
+    teams_file = locations_params[:teams]
+    game_teams_file = locations_params[:game_teams]
+
+    games_array = []
+    CSV.foreach(games_file, headers: true, header_converters: :symbol) do |row|
+      games_array << row
```

Comment:
Right now the data is stored in arrays, where each element is a row.  Didn't want to go ahead and create classes yet so just created arrays of each row, from CSV files.